content/about/projects/pangeo.md

```diff
@@ -0,0 +1,17 @@
++++
+# Homepage
+type = "blank"
+headless = true  # Homepage is headless, other widget pages are not.
+weight = 41
+title = "Pangeo"
+
+[design]
+  columns = "2"
++++
+
+{{< figure src="http://pangeo.io/_images/pangeo_simple_logo.svg" class="projects-image" >}}
```

Comment:
This src is triggering a complaint from netlify about mixed content, can you switch both of these over to https://pangeo.io?

Reply:
ah thanks for catching this! I'll merge it in and we can iterate further